packages/timeline-state-resolver-types/src/hyperdeck.ts

```diff
@@ -82,8 +82,16 @@ export interface TimelineObjHyperdeckTransport extends TimelineObjHyperdeck {
 		deviceType: DeviceType.HYPERDECK
 		type: TimelineContentTypeHyperdeck.TRANSPORT
 
-		/** The status of the hyperdeck. To start a recording, set to TransportStatus.RECORD */
+		/** The status of the hyperdeck. To start a recording, set to TransportStatus.RECORD. To start playback, set to TransportStatus.PLAY. */
```

Comment:
I think at this point we should create a typescript interface for each transport state, each with the correct properties and then `TimelineObjHyperdeckTransport` should be a union type of these interfaces. This way when somehow uses `TransportStatus.Play` they will not get type completions for recordFilename etc.